clients/shared/utils.go

```diff
@@ -25,7 +25,7 @@ func BackfillColumn(cfg config.Config, dwh destination.DataWarehouse, column col
 	}
 
 	additionalDateFmts := cfg.SharedTransferConfig.TypingSettings.AdditionalDateFormats
-	defaultVal, err := column.DefaultValue(&columns.DefaultValueArgs{Escape: true, DestKind: dwh.Label()}, additionalDateFmts)
+	defaultVal, err := column.DefaultValue(dwh.Dialect(), additionalDateFmts)
```

Comment:
This is the only place we call `DefaultValue` so I removed the `Escape` argument.